templates/default/rt/rtmessageadd.html

```diff
@@ -201,6 +201,7 @@ <H1>{$layout.pagetitle}</H1>
 					{foreach $_RT_PRIORITIES as $priorityidx => $priority}
 					<OPTION value="{$priorityidx}"{if $message.priority == $priorityidx} selected{/if}>{$priority}</OPTION>
 					{/foreach}
+					<OPTION value="" {if !isset($message.priority) || is_null($message.priority)} selected{/if}>{trans("undefined")}</OPTION>
```

Comment:
jeśli zmienna ma wartość null to isset() zwraca true

Reply:
Zauważ, że tutaj warunkiem jest OR - czyli przy null nadal zostanie spełniony cały warunek.

Reply:
https://gist.github.com/interduo/bcc504a5900cd8c73b4faccb3e2428d8
W skrypcie testowym **true** jest dla przypadku 2 i 5 - czyli tak jak chcemy.

Reply:
To przygotuj jeszcze skrypt testowy, który przetestuje to co trzeba ;-)

Reply:
Nie widzę błędu - więc stestowałem wszystkie przypadki.

Reply:
Ok - faktycznie mogę zostawić samo !isset o to Ci chodziło! ;-)

Reply:
Optymalizacje w cenie! :)

Reply:
Podstawą zapamiętać na wieki, że zmienna o wartość null w isset() zwraca false, czyli null jest traktowane tak jak by zmiennej nie było. Możesz usunąć zmienną albo przez unset(), albo przez nadanie wartości null!